src/modules/file-explorer/actions/file-explorer.js

```diff
@@ -2,31 +2,56 @@ import axios from 'axios';
 import {FileService} from '../../core/helpers';
 import {Permissions, Server, Notification} from '../../core/helpers/';
 
-const saveFile = (url, fileName) => {
-	var options = {
-		responseType: 'blob'
-	};
-
-	axios.get(url, options).then(res => {
-		var a = document.createElement('a');
-		a.href = window.URL.createObjectURL(res.data);
-		a.download = fileName;
-		a.style.display = 'none';
-		document.body.appendChild(a);
-		a.click();
-	});
+window.saveFile = function (url, fileName) {
+		//iOS devices not supported
+	if (/(iP)/g.test(navigator.userAgent)) {
+		alert('Your device does not support files downloading. Please try again in desktop browser.');
```

Comment:
I thin it would be better, if we show notifications in german. You also can use Notification.showError("message") or showInfo :)